test/mitmproxy/addons/test_dumper.py

```diff
@@ -28,43 +28,40 @@ def test_configure():
 
 
 def test_simple():
-    d = dumper.Dumper()
+    sio = io.StringIO()
+    d = dumper.Dumper(sio)
     with taddons.context(options=dump.Options()) as ctx:
-        sio = io.StringIO()
-        ctx.configure(d, tfile = sio, flow_detail = 0)
+        ctx.configure(d, flow_detail = 0)
```

Comment:
Please remove the whitespace: `..., kwarg=value, ...`
We only accept whitespace if there is a long list of keyword-args, written one per line.